new/brewblox_service/announcer.py

```diff
@@ -34,22 +22,22 @@
         'name': name,
         'active': True,
         'proxy': {
-            # Strips 'listen_path'
-            'strip_path': True,
+            # listen path is part of actual endpoint
```

Comment:
Is this reworded correctly? path on this service is equal to path on gateway. The gateway only proxies on host:port level and does not rewrite url/endpoint.

Reply:
Yes. Example:
* Gateway url = `http://gateway:1234`
* Controller url = `http://controlhost:5678`
* Controller service name = `my_controller`

`http://gateway:1234/my_controller/api/doc` will now proxy to `http://controlhost:5678/my_controller/api/doc`. 

The gateway `listen_path` is `my_controller`. This should also be part of the proxied-to url.

Reply:
example makes it very clear, can be copied directly to code comments